docs/pages/versions/unversioned/sdk/crypto.md

```diff
@@ -69,7 +69,7 @@ import * as Crypto from 'expo-crypto';
 
 ## Error Codes
 
-| Code                     | Description                                                                                                                                                     |
-|--------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `ERR_CRYPTO_UNAVAILABLE` | __Web Only.__ Access to the WebCrypto API is restricted to secure origins (https). <br/>You can run your web project from a secure origin with `expo start --https`. |
-| `ERR_CRYPTO_DIGEST`      | An invalid encoding type provided.                                                                                                                              |
+| Code                     | Description                                                                                                                                                              |
+| ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `ERR_CRYPTO_UNAVAILABLE` | **Web Only.** Access to the WebCrypto API is restricted to secure origins (https). <br/>You can run your web project from a secure origin with `npx expo start --https`. |
```

Comment:
We have pills/labels for this now (web only), I think?

Reply:
https://github.com/expo/expo/blob/main/docs/ui/components/Tag/PlatformTag.tsx

Reply:
I guess we can address this in a follow up PR?

Reply:
cc @EvanBacon - looks like you missed @kbrandwijk's comment here - should use the platform tag like @Simek linked to